### DISCUSSION TITLE
Help with concrete analysis milestone definition

### DISCUSSION THREAD

**Discussion by kdblocher** [kdblocher/bridge#37] [Ideas]
I am struggling with coming up with a clear definition for "basic analysis" milestone. We've generated a lot of good ideas so far but my engineering experience tells me we need to keep our eye on the goal, so it's time to try and define what the first goal should be to help on iterating on critical user stories.

Current definition is here: https://github.com/kdblocher/bridge/milestone/1

**Comment by eieioAce** [comment]
There are multiple types of questions, and different types of questions need different types of engineering solution. We need to categorize questions and prioritize them based on engineering resources.

**Comment by eieioAce** [comment]
One type of questions is like, given 1S opening, splinter from responder (with a point range), when is it sufficient for opener to proceed. This kind of questions need DDS, but no system analysis.

**Reply by kdblocher** [reply]
Seems like hooking up DDS is highest priority for now.

**Comment by eieioAce** [comment]
If we need to evaluate two different conventions, for instance, longer suit game try vs. short suit game try, then we can't just use DDS. The main difference is the information exposed to defender, and it can induce some defensive errors. In this case, we need some bots to play thousands of boards for us to simulate how hiding information can help.

**Reply by kdblocher** [reply]
If the convention changes the resulting contract, we can judge it. If it doesn't, we would need an inference engine of some sort.
